webapp/components/wpt-results.html

```diff
@@ -237,9 +237,11 @@
                 </template>
                 <template is="dom-if" if="[[diffShown]]">
                   <td class$="numbers [[ testResultClass(node, index, diffRun, 'passes') ]]">
-                    <span class$="passes [[ testResultClass(node, index, diffRun, 'passes') ]]">{{ getNodeResultDataByPropertyName(node, -1, diffRun, 'passes') }}</span>
+                    <span class="delta passes">{{ getNodeResultDataByPropertyName(node, -1, diffRun, 'passes') }}</span>
```

Comment:
Maybe class name "improvements" to match the documentation?

Reply:
I'm re-using the existing styling; the less changes the better.

Reply:
Oh, didn't realize it was an existing class.